node_modules/col-assets/lib/api.js

```diff
@@ -513,7 +517,25 @@ var getAssets = module.exports.getAssets = function(ctx, filters, sort, limit, o
           });
         });
 
-        return callback(null, data);
+        var options = {
+          'where': {
+            'asset_id': assetIds
+          }
+        };
+        DB.Pin.findAll(options).complete(function(err, pins) {
```

Comment:
@johncrossman I might be off on this, but I think this additional query might not be necessary since we already have an asset-to-pins join specified in the big old raw query for assets. See here:
https://github.com/sequelize/sequelize/issues/1830
(toward the bottom of the thread, where answers start coming), for the right magic options to have Sequelize parse a raw query into models with associations.